testing/tests/DevExpress.ui.widgets/tabs.tests.js

```diff
@@ -656,6 +658,30 @@ QUnit.module('Horizontal scrolling', () => {
         assert.ok(itemOffset <= rightBoundary, `item offset ${itemOffset} is lower than right boundary ${rightBoundary}`);
         assert.ok(itemOffset > contentLeft, `item offset ${itemOffset} is greater than left boundary ${contentLeft}`);
     });
+
+    QUnit.test('tabs should scroll to the disabled item when it have focus', function(assert) {
+        const items = [{ text: 'item 1' }, { text: 'item 2' }, { text: 'item 3', disabled: true }];
+        const $element = $('#scrollableTabs').dxTabs({
+            items: items,
+            scrollingEnabled: true,
+            visible: true,
+            width: 200,
+            showNavButtons: false,
+            focusStateEnabled: true,
+        });
+        const $item = $element.find('.' + DISABLED_STATE_CLASS).eq(0);
+        const keyboard = keyboardMock($element);
+
+        keyboard.press('right');
+        keyboard.press('right');
+
+        const contentLeft = Math.round($element.offset().left);
+        const contentRight = Math.round(contentLeft + $element.outerWidth());
+        const itemLeft = Math.round($item.offset().left);
+        const itemRight = Math.round(itemLeft + $item.outerWidth());
+
+        assert.strictEqual(itemRight - contentRight, 0, 'focused item in disabled state in view');
```

Comment:
It's not necessary to copypaste `Math.round` for every value. We can just use a `roughEqual` instead of a `strictEqual`
```suggestion
        assert.roughEqual(itemRight, contentRight, 1, 'focused disabled item is in view');
```

Reply:
Thanks a lot, this method really looks better. I fixed it here [87ee87bc1c61b02d65c6472f29438a5a1611e46f].